src/NTokenGeometricMean/NTokenGeometricMeanMath.sol

```diff
@@ -119,7 +114,7 @@ function computeNextLiquidity(
         uint256 a =
             uint256(int256(reserves[i]).powWad(int256(params.weights[i])));
         if (accumulator != 0) {
-            accumulator.mulWadUp(a);
+            accumulator = accumulator.mulWadUp(a);
```

Comment:
You could use the pattern of initializing `accumulator` to `ONE` in this function, too.